src/panels/lovelace/mixins/element-click-mixin.js

```diff
@@ -12,8 +13,78 @@ import computeStateName from "../../../common/entity/compute_state_name";
 export default dedupingMixin(
   (superClass) =>
     class extends NavigateMixin(EventsMixin(superClass)) {
-      handleClick(hass, config) {
-        const tapAction = config.tap_action || "more-info";
+      registerMouse(config) {
+        let ripple = null;
+
+        const createRipple = () => {
+          if (ripple === null) {
+            ripple = document.createElement("mwc-ripple");
+            this.shadowRoot.querySelector("div").appendChild(ripple);
+            ripple.setAttribute("unbounded", "");
+          }
+        };
+
+        const startAnimation = () => {
+          if (ripple) {
+            ripple.setAttribute("primary", "");
+            ripple.setAttribute("active", "");
+          }
+        };
+        const stopAnimation = () => {
+          if (ripple) {
+            ripple.removeAttribute("primary");
+            ripple.removeAttribute("active");
+            ripple.parentNode.removeChild(ripple);
```

Comment:
I think that instead of removing it, we can just set the attribute `disabled`. Then we don't have to remove all the other attributes either.

Reply:
I thought so too, but if it's disabled, it just stops animating, i.e. doesn't disappear.